src/App.tsx

```diff
@@ -1,61 +1,122 @@
+import React, { useState } from 'react';
 import './App.scss';
+import { TodoList } from './components/TodoList';
+import { TodoWithPerson, Person } from './type';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+function getUser(userId: number): Person | null {
+  const foundUser = usersFromServer.find(user => user.id === userId);
+
+  return foundUser || null;
+}
+
+const todos: TodoWithPerson[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUser(todo.userId),
+}));
+
+export const App:React.FC = () => {
+  const [todosList] = useState([...todos]);
+  const [addTodoTitle, setTodoTitle] = useState('');
+  const [personID, setPerson] = useState(0);
+  const [setNotValidPerson, setErrorUser] = useState(false);
+  const [errorTittle, setErrorTittle] = useState(false);
```

Comment:
addTodoTitle and setNotValidPerson were a bit confusing for me because they sounded like action verbs and not the names of variables. I would rename them for smth like NewTodoTitle and invalidUser... smth like that